src/controller/CHIPDeviceController.cpp

```diff
@@ -3206,6 +3216,28 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         );
     }
     break;
+    case CommissioningStage::kICDSendStayActive: {
+        if (!(params.GetICDStayActiveDurationMsec().HasValue()))
+        {
+            CommissioningStageComplete(CHIP_NO_ERROR);
+            return;
+        }
+
+        IcdManagement::Commands::StayActiveRequest::Type request;
+
+        request.stayActiveDuration = params.GetICDStayActiveDurationMsec().Value();
+        ChipLogError(Controller, "Send ICD StayActive with Duration %u", request.stayActiveDuration);
+        CHIP_ERROR err =
+            SendCommissioningCommand(proxy, request, OnICDManagementStayActiveResponse, OnBasicFailure, endpoint, timeout);
```

Comment:
Which object is "proxy" here?  Is this using PASE or CASE?

Reply:
it is using CASE.
See the below
    case CommissioningStage::kFindOperational:
        return CommissioningStage::kICDSendStayActive;
    case CommissioningStage::kICDSendStayActive:

Reply:
@yunhanw-google But how does that make us use CASE?  I don't think it does, on its own.

Reply:
Just update the pull request with kFindOperational/kICDSendStayActive, kFindOperational/kSendComplete, if kICDSendStayActive is not required, we skip kFindOperational and go with kSendComplete
When pairing with LIT and icd-stay-active-duration is et, confirmed the first CASE for StayActive has gone away.